module-2/lab-matplotlib-seaborn/your-code/challenge-2.ipynb

```diff
@@ -23,7 +23,7 @@
   },
```

Comment:
- Here 
```Python 
x = fitbit_months['Months']
y = fitbit_months['Steps']
plt.plot(x, y)
```
You should have played with the months_encoded and the xticks to set the month name on your x axis. It could be something like the following
```Python

# Median Steps by Month_encoded
weekday_steps = fitbit['Steps'].groupby(fitbit['Months_encoded']).median()
weekday_steps.head(5)

fig,axes = plt.subplots(figsize = (12,4),nrows = 1, ncols = 2)
plt.sca(axes[0])
weekday_steps.plot(kind = 'line',linewidth=2)
plt.ylabel('Median number of steps')
plt.xlabel('Months')
plt.title('Median Steps walked by month')
plt.xticks(list(range(13)),['','Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec'])
```
- It should be better to set as y variable steps instead of minutes as you have written down in your code. 
```Python
x = ['Steps']
y = ['Minutes Lightly Active', 'Minutes Very Active', 'Minutes Sedentary']
```